lib/stitches/version.rb

```diff
@@ -1,3 +1,3 @@
 module Stitches
-  VERSION = '3.7.2'
+  VERSION = '3.7.3'
```

Comment:
Don't need to do this no the branch - the PR template should outline the process, but you can bump this after you merge to master.